### PR TITLE
USWDS - Redirects: Fix 'Components' link in page previews

### PR DESCRIPTION
**Pull request by mejiaj**
## Description

**Fixed an issue with redirects not working properly.**  Removed default `url` setting in Jekyll's `_config.yml` that was messing with redirects in Federalist. For example, previously if you clicked on `Components` in federalist preview, the redirect to `/components/overview` would return a 404.

**Updated USWDS Federalist preview URL**. The `federalist_base` now accurately reflects the new pages.cloud.gov url for USWDS components. The migration from federalist to pages.cloud.gov was done on 09/26/22. 

## Additional information

Additionally preview images are now working properly. For example, when pasting a USWDS link to Slack.

**Before**
image

image

**After**
image

image

## How to test
- [ ] Visit [preview link →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/jm-fix-component-preview/)
- [ ] Click on a link that has a redirect ex: `Components` link
- [ ] You should **should** be redirected to `/components/overview`

---

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [x] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [x] Run `npm test` and make sure the tests for the files you have changed have passed.
- [ ] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [x] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -4,11 +4,9 @@ description: USWDS makes it easier to build accessible, mobile-friendly governme
 google_analytics_ua: UA-48605964-43
 uswds_version: 3.1.0
 uswds_email: uswds@gsa.gov
-federalist_base: "https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds"
+federalist_base: "https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds"
```
Comment (mejiaj) [inline _config.yml:7]: Future-proof; updates URL because of Federalist migration to `pages.cloud.gov`.

File: _config.yml [hunk 2]
```diff
@@ -79,6 +77,10 @@ collections:
     permalink: /:collection/:title
 
 defaults:
+  - scope:
+      path: ""
+    values:
+      image: /img/uswds-logo/lg-black.png
```
Comment (mejiaj) [inline _config.yml:83]: It made more sense to move fallback image up since it's expected to be overridden.

File: _config.yml [hunk 3]
```diff
@@ -148,9 +146,9 @@ exclude:
   - css
   - pa11y-results.json
 
-env: dev
-
-url: https://designsystem.digital.gov
+# Jekyll SEO tag
 twitter:
   username: USWDS
   card: summary_large_image
```
Comment (mejiaj) [inline _config.yml:152]: Re-ordered and added comment to better contain SEO plugin specific settings. Hoping it will improve clarity for other devs.

**Review by thisisdano — APPROVED** [review]
This is a nice fix that addresses a few items I wouldn't have expected to be related

**Review by amyleadem — CHANGES_REQUESTED** [review]
The redirects are working! Nice solution. 

For the images, I did notice that the meta didn't  seem to be updating as expected on the /next page:
Slack preview: 
MicrosoftTeams-image (4)

In the source, og:image and twitter:image are still showing lg-black.png

MicrosoftTeams-image (3)

Let me know if you have questions.

**Comment by mejiaj** [comment]
@amyleadem I suspect that was a caching issue on my end. I've since updated the old `og:image` key to use the SEO gem's recommended `image` key.

```diff
# _next/02-introduction.md
- meta:
-  og:image: /img/next/og-next-report.png
+ image: /img/next/og-next-report.png
```

Can you try again?

**Review by amyleadem — APPROVED** [review]
The metadata looks good! Redirects look good. Nice work!

Slack preview:
image

**Pull request merged**